src/pretix/presale/templates/pretixpresale/event/fragment_product_list.html

```diff
@@ -228,7 +228,7 @@ <h5 id="item-{{ item.pk }}-{{ var.pk }}-legend">{{ var }}</h5>
                     </div>
                 </article>
             {% else %}
-                <article aria-labelledby="item-{{ item.pk }}-legend"{% if item.description %} aria-describedby="item-{{ item.pk }}-description"{% endif %} class="row product-row simple" id="item-{{ item.pk }}">
+                <article aria-labelledby="item-{{ item.pk }}-legend"{% if item.description %} aria-describedby="item-{{ item.pk }}-description"{% endif %} class="row product-row simple" id="item-{{ item.pk }}"{% if not item.free_price %} data-price="{{ item.price }}"{% endif %}>
```

Comment:
`item` hat keinen `price` sondern nur einen `default_price`

```suggestion
                <article aria-labelledby="item-{{ item.pk }}-legend"{% if item.description %} aria-describedby="item-{{ item.pk }}-description"{% endif %} class="row product-row simple" id="item-{{ item.pk }}"{% if not item.free_price %} data-price="{{ item.default_price }}"{% endif %}>
```

Reply:
Doh, did not realize this as my only test-product, that was not either free_price or a variation-product, was free … Thanks!